### PR TITLE
feat: add styled components

### PR DESCRIPTION
**Pull request by andrewlitt**
# Description

Installed [styled-components](https://styled-components.com/) as a CSS-in-JS framework.

Converted App.js to styled-components. Deleted the App.css file.

Converted index.js to styled-componens by using the createGlobalStyle function, allowing us to assign css to the `body` tag. Deleted the index.css file.

## Related Issue

None.

## Motivation and Context

Setting up our frontend to use styled-components for styling. Taking a CSS-in-JS approach.

## How Has This Been Tested

Running on localhost works. Verified global styles injected into body tag.

## Screenshots (if appropriate)

Nah dawg

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -1,23 +1,53 @@
-import React from "react";
-import logo from "./chef.png";
-import "./App.css";
+import React from 'react';
+import logo from './chef.png';
+import styled, { keyframes } from 'styled-components'
```
Comment (zackharley) [inline web/src/App.js]: Missing a semi-colon

File: web/src/App.js [hunk 2]
```diff
@@ -1,23 +1,53 @@
...
+  background-color: #282c34;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+`
+const AppLogoSpin = keyframes`
```
Comment (zackharley) [inline web/src/App.js:18]: Ou, this is cool

**Review by zackharley — DISMISSED** [review]
One teeny comment, but looks good to me!!

File: web/src/App.js (hunk 1 above)
Comment (andrewlitt) [reply web/src/App.js]: fixed!

File: web/src/App.js (hunk 2 above)
Comment (andrewlitt) [reply web/src/App.js:18]: I know right! Gonna feel weird for a bit, and gotta minimize these to keep the editor clean, but I feel it'll be cool

**Review by ColinLMacLeod1 — CHANGES_REQUESTED** [review]
This all looks good, just squash it into one commit and it's good to go

**Review by 10kc-zack — DISMISSED** [review]

**Review by lucas-manuel — DISMISSED** [review]

**Review by lucas-manuel — APPROVED** [review]

**Review by ColinLMacLeod1 — APPROVED** [review]

**Pull request merged**